api.swagger.yaml

```
@@ -6560,6 +6717,165 @@ components:
+              description: 'This is the timestamp (in seconds) of when the blog was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the blog was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the blog.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the blog.'
+              type: integer
+              example: 5
+            published:
+              description: 'Published status of the blog.'
+              type: boolean
+              example: true
+    blog-schema-update:
+      allOf:
+        - $ref: '#/components/schemas/blog-schema-common'
+        - type: object
+          properties:
+            image:
+              description: 'The image for this blog.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the image.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            general_topic:
+              description: 'The genetal topic of this blog.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the genetal topic.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+    blog-schema-create:
+      allOf:
+        - $ref: '#/components/schemas/blog-schema-common'
+        - type: object
+          properties:
+            image:
+              description: 'The image for this blog.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the image.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            general_topic:
+              description: 'The general topic of this blog.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the general topic.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+          required:
+            - title
+            - body
+            - state
+    blog-schema-common:
+      type: object
+      properties:
+        title:
+          description: 'Title of the blog.'
+          type: string
+          example: 'Guidelines for creating a strong password'
+        body:
+          description: 'The body of the blog.'
+          type: string
+          example: 'You can create strong passwords using an online random password generator.'
```

```suggestion
          example: 'You can create strong passwords using an offline random password generator.'
```
sorry, couldn't resist